README.md

```diff
@@ -1,3 +1,18 @@
+<p align="center">
+  <a href="https://pennylane.ai/blog/2021/07/pennylane-code-together/">
+    <img width=50% src="https://pennylane.ai/blog/images/code_together.jpg">
+  </a>
+</p>
+
+<p align="center">
+  <strong>Announcing <a href="https://pennylane.ai/blog/2021/07/pennylane-code-together/">PennyLane: Code Together</a>!
+  Join us on GitHub August 16th-27th, see <a href="https://github.com/PennyLaneAI/pennylane/blob/master/code_together.md">event FAQ here</a></strong>.
+</p>
+
+<br>

```

Comment:
Maybe might look slightly better with a horizontal line here, to separate the callout from the rest of the readme?

![image](https://user-images.githubusercontent.com/2959003/128295276-68adf584-096d-4da9-96aa-d878d310ab82.png)


```suggestion
---
```